tests/nlu/featurizers/test_spacy_featurizer.py

```diff
@@ -1,23 +1,33 @@
 import numpy as np
 import pytest
 
+from rasa.nlu.utils.spacy_utils import SpacyModel, SpacyModelProvider, SpacyPreprocessor
 from rasa.shared.nlu.training_data import loading
 from rasa.shared.nlu.training_data.training_data import TrainingData
 from rasa.shared.nlu.training_data.message import Message
-from rasa.nlu.config import RasaNLUModelConfig
-from rasa.nlu.featurizers.dense_featurizer.spacy_featurizer import SpacyFeaturizer
+from rasa.nlu.featurizers.dense_featurizer.spacy_featurizer import (
+    SpacyFeaturizerGraphComponent,
+)
 from rasa.nlu.constants import SPACY_DOCS
 from rasa.shared.nlu.constants import TEXT, INTENT, RESPONSE
 
 
 @pytest.fixture(scope="module")
-def spacy_nlp_component(component_builder, blank_config):
-    spacy_nlp_config = {"name": "SpacyNLP", "model": "en_core_web_md"}
-    return component_builder.create_component(spacy_nlp_config, blank_config)
+def spacy_model() -> SpacyModel:
+    return SpacyModelProvider.create(
+        {"model": "en_core_web_md"}, None, None, None
```

Comment:
nit: seems a bit evil to pass `None` if `None` isn't officially allowed, what do you think?

Reply:
yeah true, I can just use the `__init__` instead.

Reply:
yeah true, it's just annoying have to pass the fixtures around everywhere.